Container.py

```diff
@@ -167,35 +182,35 @@ def add(output_register: GeneralRegister, input_register1: GeneralRegister, inpu
     if not progressive_beams:
         return []
     cstate_manager_global = GAMES[game_version]["cstate_manager_global"]
-    charge_beam_offset = 0x7F

```

Comment:
Is this the same address but essentially reading the byte rather than the bit?

Reply:
you mean reading as int rather than byte?

Reply:
and yes it is the same thing we're reading from